chart/templates/secrets.yaml

```diff
@@ -28,10 +28,10 @@ type: Opaque
 stringData:
   POSTGRES_USER: {{ $user | quote }}
   POSTGRES_PASSWORD: {{ $password | quote }}
-  POSTGRES_HOST: {{ $host | quote }}
+  POSTGRES_HOST: {{ $postgresHost | quote }}
+  POSTGRES_PORT: {{ $postgresPort | quote }}
   SSLMODE: {{ $sslmode | quote }}
```

Comment:
I would not change the DB_URL secret, keep it as is and and another value into this secret with PGBOUNCER_DB_URL - This URL would be static e.g. `host=pgbouncer` and then the MTLS cert needs be mounted in the pods, with the PGSSLCERT and PGSSLKEY environment variables pointing to the mounts.